CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 -   Fixed chip mixin to use correct accent color issue ([#72](https://github.com/etn-ccis/blui-angular-themes/issues/72)).
 -   Fixed `<blui-inline>` does not work with inline svg icons issue ([#133](https://github.com/etn-ccis/blui-angular-themes/issues/133)).
 -   Fixed Dark theme `<blui-drawer-header>` text and icon color issue ([#79](https://github.com/etn-ccis/blui-angular-themes/issues/79)).
+-   Fixed Remove show password icon in the Edge browser issue ([#137](https://github.com/etn-ccis/blui-angular-workflows/issues/137)).
```

Comment:
This still does not read like a sentence to me. And you did not update the issue number either.

```suggestion
-   Fixed an issue where password field displays two show password icons on the Edge browser ([#168](https://github.com/etn-ccis/blui-angular-themes/issues/168)).
```

Reply:
This now reads as if a default "password" input uses 2 icons when it doesn't. This issue arose from the auth workflow where an additional icon is provided in the password input field. This should probably read something like "Fixed an issue where the browser default password reveal button collides with any matSuffix input icon provided"

Also, do we want to do this for all inputs? I'm not sure we ever want someone to use the browser's default password reveal button, but want to make sure we are sure we want this to be all-encompassing.

Reply:
@daileytj Updated the sentences as per the your suggest in the CHANGELOG.md file. 
@huayunh Thanks for updated the issue number and link